scripts/docker/web/base/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# base image for elektra web, all other images build upon this (builds elektra with `yajl` and `kdb`)
+
+FROM ubuntu:16.04
+
+# elektra deps
+RUN apt-get update -y
+RUN apt-get install -y cmake git build-essential
+
+# elektra web deps
+RUN apt-get install -y libyajl-dev curl nodejs-legacy npm
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+
+# pull latest libelektra
+RUN mkdir /home/elektra
+WORKDIR /home/elektra
+RUN git clone --depth 1 https://github.com/ElektraInitiative/libelektra.git
```

Comment:
You should pin a version for reproducible builds.

Reply:
you did this before by checking out the tag. I would go for a build-arg that you feed to a --branch parameter of clone (you can let it default to master if you want).

Reply:
@ingwinlu good idea, I did that now!